clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/model/AmazonInstance.groovy

```diff
@@ -88,6 +88,10 @@ class AmazonInstance implements Instance, Serializable {
     getExtraAttributes().get("placement")?.availabilityZone
   }
 
+  String getAvailabilityZone() {
```

Comment:
this just gets included as part of `ServertGroup`?

Reply:
No, this was missing from the `instance` object when expanded vs. not expanded because the non expanded returns a `InstanceViewModel` which maps `zone` -> `availabilityZone` whereas expanded just returns the entire `instance` object. 🤷‍♂️